src/main/java/mate/academy/bookstore/dto/CreateBookRequestDto.java

```diff
@@ -1,13 +1,20 @@
 package mate.academy.bookstore.dto;
 
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotNull;
 import java.math.BigDecimal;
 import lombok.Data;
 
 @Data
 public class CreateBookRequestDto {
+    @NotNull
     private String title;
+    @NotNull
     private String author;
+    @NotNull
     private String isbn;
```

Comment:
we can add `@ISBN` to check its correctness

```suggestion
    @NotNull
    @ISBN
    private String isbn;
```